src/test/java/Entities/BoardTest.java

```diff
@@ -0,0 +1,163 @@
+        List<Player> players = new ArrayList<Player>();
+        players.add(playerOne);
+        players.add(playerTwo);
+        int [] rentValues1 = new int[] {1,2,3,4,5};
+        int [] rentValues2 = new int[] {2,4,6,8,10};
+        Property cellOne = new Property("cellOne", "Colour", 0, 0, rentValues1, playerOne, 0,
+                0, false);
+        Property cellTwo = new Property("cellTwo", "Colour", 0, 0, rentValues2, playerTwo, 0,
+                0, false);
+        List<Cell> cells = new ArrayList<Cell>();
+        cells.add(cellOne);
+        cells.add(cellTwo);
+        Board board = new Board(players, cells);
+        playerOne.move(3);
+        playerTwo.move(2);
+        board.updatePlayerPosition(playerOne);
+        board.updatePlayerPosition(playerTwo);
+        Map<Player, Integer> actual = board.getPlayerPositions();
+        Map<Player, Integer> expected = new HashMap<Player, Integer>();
+        expected.put(playerOne, 3);
+        expected.put(playerTwo, 2);
+        assertEquals(actual, expected);
+    }
+
+    @Test
+    public void testGetPlayerCell(){
+        Player playerOne = new Player("PlayerOne", 0, false, 0, 1);
+        Player playerTwo = new Player("PlayerTwo");
+        List<Player> players = new ArrayList<Player>();
+        players.add(playerOne);
+        players.add(playerTwo);
+        int [] rentValues1 = new int[] {1,2,3,4,5};
+        int [] rentValues2 = new int[] {2,4,6,8,10};
+        Property cellOne = new Property("cellOne", "Colour", 0, 0, rentValues1, playerOne, 0,
+                0, false);
+        Property cellTwo = new Property("cellTwo", "Colour", 0, 0, rentValues2, playerTwo, 0,
+                0, false);
+        List<Cell> cells = new ArrayList<Cell>();
+        cells.add(cellOne);
+        cells.add(cellTwo);
+        Board board = new Board(players, cells);
+        Cell actual = board.getPlayerCell(playerOne);
+        Cell expected = cells.get(playerOne.getPosition());
+        assertEquals(actual, expected);
+    }
+
+    @Test
+    public void testUpdatePlayerPosition(){
+        Player playerOne = new Player("PlayerOne");
+        Player playerTwo = new Player("PlayerTwo");
+        List<Player> players = new ArrayList<Player>();
+        players.add(playerOne);
+        players.add(playerTwo);
+        int [] rentValues1 = new int[] {1,2,3,4,5};
+        int [] rentValues2 = new int[] {2,4,6,8,10};
+        Property cellOne = new Property("cellOne", "Colour", 0, 0, rentValues1, playerOne, 0,
+                0, false);
+        Property cellTwo = new Property("cellTwo", "Colour", 0, 0, rentValues2, playerTwo, 0,
+                0, false);
+        List<Cell> cells = new ArrayList<Cell>();
+        cells.add(cellOne);
+        cells.add(cellTwo);
+        Board board = new Board(players, cells);
+        playerOne.move(3);
+        board.updatePlayerPosition(playerOne);
+        assert board.playerPositions.get(playerOne) == 3;
+    }
+
+}
```

Comment:
Great tests overall. For future implementations, it maybe helpful to create a helper function that initialize new instances for you instead of having to type out the whole thing for each tests. You can check out PlayerTest to see an example. Otherwise, great job!